cms/templates/import.html

```diff
@@ -102,8 +102,8 @@ <h3 class="title">
           <ol class="status-progress list-progress">
             <li class="item-progresspoint item-progresspoint-upload is-complete">
               <span class="deco status-visual">
-                <i class="icon fa fa-cog"></i>
-                <i class="icon fa fa-warning"></i>
+                <span class="icon fa fa-cog" aria-hidden="true"></span>
```

Comment:
Do you think sr-text is necessary? Next to the icons there's text that either says "Uploading" or "Uploaded" when it's done, so it seems like the icons are just there to be decorative.